patterns/atoms/buttons/primary.php

```diff
@@ -1,2 +1,17 @@
-<p><a href="#" class="a__btn a__btn__primary">Click Me!</a></p>
-<p><a href="#" class="a__btn a__btn__primary" disabled="disabled">Disabled</a></p>
+<?php
```

Comment:
The code is part of the old implementation with twig so feel free to change anything since most of it is not relevant.